kolibri/core/device/management/commands/provisiondevice.py

```diff
@@ -73,8 +32,19 @@ def get_user_response(prompt, valid_answers=None, to_lower_case=True):
 languages = dict(settings.LANGUAGES)
 
 
-def create_facility(facility_name=None, preset=None, interactive=False):
-    if facility_name is None and interactive:
+def json_file_contents(parser, arg):
+    if not os.path.exists(arg) or not os.path.isfile(arg):
+        return parser.error("The file '{}' does not exist".format(arg))
+    with open(arg, "r") as f:
+        try:
+            output = json.load(f)
+        except ValueError as e:  # Use ValueError rather than JSONDecodeError for Py2 compatibility
+            return parser.error("The file '{}' is not valid JSON:\n{}".format(arg, e))
+    return output
```

Comment:
super minor - I think this behaves the same without a temporary variable?

```suggestion
            return json.load(f)
        except ValueError as e:  # Use ValueError rather than JSONDecodeError for Py2 compatibility
            return parser.error("The file '{}' is not valid JSON:\n{}".format(arg, e))
```

Reply:
Yes, this was like that when I got here - some guy called @indirectlylit made the temporary variable... :)